packages/cli/src/util/telemetry/index.ts

```diff
@@ -29,6 +29,7 @@ export class TelemetryClient {
   store: TelemetryEventStore;
 
   protected redactedValue = '[REDACTED]';
+  protected noValueToTriggerPrompt = '[TRIGGER_PROMPT]';
```

Comment:
Thinking this is a value that says we prompted the user for this value since they didn't provide it. Open to suggestions here.

Reply:
I think this, or just having it missing and we check by `sessionId` where is missing are both fine.